### PR TITLE
I789 key goals subcard

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

File: app/imports/ui/react/organization-settings/components/KeyGoals.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
 import React from 'react';
-import { WorkspaceDefaultsTypes } from '/imports/share/constants';
+import PropTypes from 'prop-types';
+import { WorkspaceDefaultsTypes, TimeScaleOptions } from '/imports/share/constants';
```
Comment (Dartv) [inline app/imports/ui/react/organization-settings/components/KeyGoals.js]: Don't use global imports

File: app/imports/ui/react/organization-settings/components/KeyGoals.js [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 import CardBlockCollapse from '/imports/ui/react/components/CardBlockCollapse';
 import LimitNumberField from './LimitNumberField';
 import CardBlock from '../../components/CardBlock';
+import FormField from '../../forms/components/FormField';
```
Comment (Dartv) [inline app/imports/ui/react/organization-settings/components/KeyGoals.js]: You can import `FormField`, `CardBlock` and `SelectInput` together. `import { CardBlock, FormField, SelectInput } from '../../components'`

File: app/imports/ui/react/organization-settings/components/KeyGoals.js [hunk 3]
```diff
@@ -15,14 +23,33 @@ const fields = [
   },
 ];
 
-const KeyGoalsSettings = props => (
+const KeyGoalsSettings = ({ changeWorkspaceDefaults, ...restProps }) => (
   <CardBlockCollapse leftText="Key goals">
     <CardBlock>
       {fields.map(field => (
```
Comment (Dartv) [inline app/imports/ui/react/organization-settings/components/KeyGoals.js]: What is the need for this? Why not just render them normally? They are just 3 static fields. This just makes code harder to read.

File: share/schemas/schemas/workspace-defaults.js [hunk 4]
```diff
@@ -16,6 +16,10 @@ export default new SimpleSchema({
       [WorkspaceDefaultsTypes.DISPLAY_ACTIONS]: options,
       [WorkspaceDefaultsTypes.DISPLAY_GOALS]: options,
       [WorkspaceDefaultsTypes.DISPLAY_COMPLETED_DELETED_GOALS]: options,
+      [WorkspaceDefaultsTypes.TIME_SCALE]: {
+        type: Number,
+        allowedValues: [1, 3, 6, 9, 12],
```
Comment (Dartv) [inline share/schemas/schemas/workspace-defaults.js]: You have a `TimeScaleOptions` constant for this. Just use `pluck` or something

**Comment by Serhiy91** [comment]
@Dartv fixed

**Pull request merged**